topology/probes/k8s/probe.go

```diff
@@ -95,3 +101,38 @@ func NewProbe(g *graph.Graph, manager string, subprobes map[string]Subprobe, lin
 		linkers:   linkers,
 	}
 }
+
+func logOnError(err error) {
+	logging.GetLogger().Warning(err)
+}
+
+type errorThrottle struct {
+	period   time.Duration
+	lastLock sync.Mutex
+	last     time.Time
+}
+
+func (r *errorThrottle) onError(error) {
+	r.lastLock.Lock()
```

Comment:
The lock is global and is held during the sleep (5 seconds, which is a lot). We should at least use an atomic.Value